CMakeLists.txt

```diff
@@ -235,13 +235,10 @@ endif()
 
 # Add an always out-of-date target to get the Swift compiler version.
 if (SWIFTC_FOUND)
-  set(SWIFT_VERSION ${LLBUILD_OBJ_DIR}/swift-version)
-  add_custom_target(
-      swiftversion ALL
-      COMMAND ${CMAKE_SOURCE_DIR}/utils/write-swift-version ${SWIFTC_EXECUTABLE} ${SWIFT_VERSION}
-      BYPRODUCTS ${SWIFT_VERSION}
-      COMMENT "Checking Swift compiler version"
-  )
+  execute_process(COMMAND ${SWIFTC_EXECUTABLE} -version
+      OUTPUT_FILE ${LLBUILD_OBJ_DIR}/swift-version
+      OUTPUT_STRIP_TRAILING_WHITESPACE)
+  add_custom_target(swiftversion DEPENDS ${LLBUILD_OBJ_DIR}/swift-version)
```

Comment:
If/when this goes in, I'll close out https://github.com/apple/swift-llbuild/pull/449

Reply:
Yeah, I am concerned that this will lead to non-null incremental builds.

Reply:
Locally testing, I don't see this:

```
S:\b\llbuild>ninja
[125/132] Linking CXX executable unittests\Ninja\NinjaTests.exe
LINK : unittests\Ninja\NinjaTests.exe not found or not built by the last incremental link; performing full link
[126/132] Linking CXX executable unittests\Basic\BasicTests.exe
LINK : unittests\Basic\BasicTests.exe not found or not built by the last incremental link; performing full link
[127/132] Linking CXX executable unittests\Core\CoreTests.exe
LINK : unittests\Core\CoreTests.exe not found or not built by the last incremental link; performing full link
[128/132] Linking CXX shared library bin\libllbuild.dll
LINK : bin\libllbuild.dll not found or not built by the last incremental link; performing full link
   Creating library lib\libllbuild.lib and object lib\libllbuild.exp
[129/132] Linking CXX executable bin\swift-build-tool.exe
LINK : bin\swift-build-tool.exe not found or not built by the last incremental link; performing full link
[130/132] Linking CXX executable bin\llbuild.exe
LINK : bin\llbuild.exe not found or not built by the last incremental link; performing full link
[131/132] Linking CXX executable unittests\BuildSystem\BuildSystemTests.exe
LINK : unittests\BuildSystem\BuildSystemTests.exe not found or not built by the last incremental link; performing full link
[132/132] Linking CXX executable unittests\CAPI\CAPITests.exe
LINK : unittests\CAPI\CAPITests.exe not found or not built by the last incremental link; performing full link

S:\b\llbuild>ninja
ninja: no work to do.
```